src/AlgebraicGeometry/Schemes/AffineSchemes/Objects/Properties.jl

```diff
@@ -707,7 +701,7 @@ end
 @doc raw"""
     is_geometrically_integral(X::AbsSpec)
 
-Return if ``X/k`` is geometrically integral.
+Test if ``X/k`` is geometrically integral.
```

Comment:
Maybe use "Check" instead of "Test" for consistency.

Reply:
Perhaps... But note that we aren't consistent here to begin with. I also looked at a bunch of Julia docstrings for `is*` function, and they also used a mixture between "Check" and "Test". If someone would like to make a decision which to use, fine by me, but I feel like that would be work for a separate PR perhaps?

In the end, I think both are equally clear, so I wouldn't bother -- IMHO both are better than "Return `true` if XYZ holds, `false` otherwise" which we currently use in a lot of places and which feels rather stilted to me (full disclaimer: I've used it myself in the past)